test/cr_filename.cc

```diff
@@ -49,15 +49,16 @@ int main(int argc, const char *argv[])
         fprintf(stderr, "[ERROR] Could not open file(%s)\n", filepath);
         exit(EXIT_FAILURE);
     }
-    close(fd);
```

Comment:
I think the reason for calling `close` here is to execute `flush`/`release` on the open file.
I think they wanted to execute `open`/`close(flush)` before executing stat.

Reply:
Reverted.